app/services/admin/onboarding_wizard.rb

```diff
@@ -139,7 +139,9 @@ def init_setup_steps(community_id)
       locales = community.locales
       community_customizations = CommunityCustomization.where(community_id: community.id).select { |cc| locales.include?(cc.locale) }
       custom_field = CustomField.find_by(community_id: community.id)
-      listing = Listing.find_by(community_id: community.id)
+      listing = Listing.where(community_id: community.id)
+                  .joins(author: :community_membership)
+                  .where('community_memberships.admin' => true).first
```

Comment:
Looks good. It's a different solution from what I had in mind but feels simpler this way. The check logic is no duplicated from here to event handler but maybe that's reasonable as it speeds up and simplifies the bootstrap data processing side vs. processing all listings.